R/namespace.R

```diff
@@ -1,13 +1,21 @@
 #' Use a basic `NAMESPACE`
 #'
-#' This `NAMESPACE` exports everything, except functions that start
-#' with a `.`.
+#' If `roxygen` is `TRUE` generates a empty `NAMESPACE` that exports nothing;
+#' you'll need to explicitly export functions with `@export`. If `roxygen`
+#' is `FALSE`, generates a default `NAMESPACE` that exports all functions
+#' except those that start with `.`.
 #'
+#' @param roxygen Are you planning to use roxygen2?
 #' @seealso The [namespace chapter](https://r-pkgs.org/namespace.html) of
 #'   [R Packages](https://r-pkgs.org).
-#'
 #' @export
-use_namespace <- function() {
+use_namespace <- function(roxygen = TRUE) {
   check_is_package("use_namespace()")
-  use_template("NAMESPACE")
```

Comment:
Should we also be deleting the `NAMESPACE` template file then?

Reply:
Oooh yes